src/core_atmosphere/physics/mpas_atmphys_control.F

```diff
@@ -493,6 +495,79 @@ subroutine physics_compatibility_check(dminfo, blockList, streamManager, ierr)
 
  end subroutine physics_compatibility_check
 
+!=================================================================================================================
+!  routine sst_update_check()
+!
+!> \brief Sea Surface Interval Update Check
+!> \author Miles Curry
+!> \date   January 2019
+!> \details
+!>  This routine checks that the user has correctly set the SST update options,
+!>  by setting the `config_sst_update` namelist option equal to `true` .AND.
+!>  by setting the `input_interal` surface stream variable in streams.atmosphere
+!>  to a valid HH:MM:SS update time.
+!>
+ subroutine sst_update_check(blockList, streamManager, ierr)
+!=================================================================================================================
+
+ use mpas_timekeeping, only : mpas_split_string
```

Comment:
It looks like the use of `mpas_split_string` is unnecessary. Perhaps it was used during development for debugging or something else? In any case, it seems safe to remove this `use` statement.